synapse/swarm/runtime.py

```diff
@@ -59,11 +59,17 @@ def _getTufosByFrom(self, by, prop, valu=None, limit=None, fromtag=None):
 
     def _stormOperLift(self, query, oper):
 
```

Comment:
not so much looser as more standard.  there was a period in the beginning where operators took arguments directly from the operator tufo dict, but to standardize ( and make a safe namespace for "out of band" oper info ) i moved all oper() tufo's to expecting {'args':[],'kwlist':[]} for their parsed input.  This is cleaning up of the 2 legacy ones that was left over.  ( also making it possible to use the long form lift() operator directly in storm rather than requiring macro syntax )